src/config.ts

```diff
@@ -14,6 +14,9 @@ const mandatoryVariables = [
   'DRIVE_GATEWAY_PASSWORD',
   'DRIVE_NEW_GATEWAY_URL',
   'DRIVE_NEW_GATEWAY_SECRET',
+  'DRIVE_GATEWAY_PUBLIC_SECRET',
+  'PAYMENTS_GATEWAY_SECRET',
```

Comment:
This environment variable is not mandatory for production 

You may do something like

```
const mandatoryProductionVariables = [
  ...
];

const mandatoryTestingVariables = [
  ...
];

const mandatoryVariables = env === 'production' ? mandatoryProductionVariables : mandatoryProductionVariables.concat(mandatoryTestingVariables);
```